### ISSUE TITLE
Gitea mirror

### ISSUE THREAD

**Issue by DeaDvey** [javalsai/lidm#25]
Make a gitea mirror, or just move fully to gitea.

**Comment by javalsai** [comment]
> Make a gitea mirror, or just move fully to gitea.

Literally why, all the workflows are here, it works fine here, people can contribute here, the AUR and Nix pkgs map to github, is more reliable and my gitea server has a russian roulette uptime.

**Comment by grialion** [comment]
I think he's talking about a mirror *on Gitea* (so this project can be accessed from Gitea too)

**Comment by javalsai** [comment]
> I think he's talking about a mirror _on Gitea_ (so this project can be accessed from Gitea too)

afaik there's also no easy way to mirror GitHub -> anywhere else, and GitHub is gonna be the main mirror for this, I'll take a look tho

**Comment by DeaDvey** [comment]
> I think he's talking about a mirror _on Gitea_ (so this project can be accessed from Gitea too)

Yeah, I meant it would be cool to have a copy on Gitea, idk not that important though

**Comment by DeaDvey** [comment]
There's a nix package? I can't see it.

**Comment by javalsai** [comment]
> There's a nix package? I can't see it.

its not on nixpkgs, I did `nix run github:javalsai/lidm` before, but now I randomly get IOT instruction fault lol, maybe double free error... idk, if you can check it on ur machine perfect.

**Comment by SpamixOfficial** [comment]
tbf we could mirror this to something like gitlab instead if no one wants to host a gitea mirror (it is very easy, I've done it before) @javalsai

**Comment by javalsai** [comment]
> tbf we could mirror this to something like gitlab instead if no one wants to host a gitea mirror (it is very easy, I've done it before) [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/80113062?u=0982eda43e7f2839a0e9d68c77745686ee07257e&amp;v=4&amp;size=48">@javalsai](https://github.com/javalsai?rgh-link-date=2025-06-23T07%3A40%3A43.000Z)

I already have my gitea server with all the tuxcord stuff. Just that I have no time to try to get this working and iirc GitHub has no easy way to mirror to other services. It's easy to mirror to GitHub, but not that much from it.

**Comment by javalsai** [comment]
@DeaDvey does this look good?

https://git.javalsai.tuxcord.net/javalsai/lidm
